draft-hsts-tracking-prevention.md

```diff
@@ -0,0 +1,256 @@
+---
+###
+# Internet-Draft Markdown Template
+#
+# Rename this file from draft-todo-yourname-protocol.md to get started.
+# Draft name format is "draft-<yourname>-<workgroup>-<name>.md".
+#
+# For initial setup, you only need to edit the first block of fields.
+# Only "title" needs to be changed; delete "abbrev" if your title is short.
+# Any other content can be edited, but be careful not to introduce errors.
+# Some fields will be set automatically during setup if they are unchanged.
+#
+# Don't include "-00" or "-latest" in the filename.
+# Labels in the form draft-<yourname>-<workgroup>-<name>-latest are used by
+# the tools to refer to the current version; see "docname" for example.
+#
+# This template uses kramdown-rfc: https://github.com/cabo/kramdown-rfc
+# You can replace the entire file if you prefer a different format.
+# Change the file extension to match the format (.xml for XML, etc...)
+#
+###
+title: "HSTS Tracking Prevention"
+category: info
+
+docname: draft-hsts-tracking-prevention-latest
+submissiontype: IETF  # also: "independent", "editorial", "IAB", or "IRTF"
+number:
+date:
+consensus: true
+v: 3
+area: AREA
+workgroup: WG Working Group
+keyword:
+ - HSTS
+ - tracking prevention
+venue:
+  group: WG
+  type: Working Group
```

Comment:
Can we put in the WG we hope that would adopt this?

Reply:
I'm not sure ATM which WG would be the best option for this. RFC6797 was originally in the websec WG but that has since concluded.

Removed this section for now.